.github/dependabot.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      actions:
+        patterns:
+          - "*"
```

Comment:
this means you will always bump every action right? I don't think its useful

Reply:
It will open PR with changes like this:
![CleanShot 2023-10-05 at 11 40 36@2x](https://github.com/saleor/macaw-ui/assets/9116238/09f3eaa3-4cbb-41e7-be37-930680b4ab32)

So bot will update actions like `checkout` or `pnpm/setup` and I think we want that 😄

Reply:
Do you want to update all or nothing? Why don't you want 5 independent PRs? One of five can break your CI